web/src/features/Gitops/GitOpsDeploymentManager.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useContext } from "react";
+import find from "lodash/find";
+import Loader from "../../components/shared/Loader";
+import ErrorModal from "../../components/modals/ErrorModal";
+import { withRouter } from "react-router-dom";
+import { requiresHostname } from "../../utilities/utilities";
+import { Flex, Paragraph } from "../../styles/common";
+import SetupProvider from "./components/SetupProvider";
+import AppGitops from "./components/AppGitops";
+import { GitOpsContext, withGitOpsConsumer } from "./context";
+import {
+  SERVICES,
+  BITBUCKET_SERVER_DEFAULT_SSH_PORT,
+  BITBUCKET_SERVER_DEFAULT_HTTP_PORT,
+} from "./constants";
+import "../../scss/components/gitops/GitOpsDeploymentManager.scss";
+
+const STEPS = [
+  {
+    step: "provider",
+    title: "GitOps Configuration",
+  },
+  {
+    step: "action",
+    title: "GitOps Configuration ",
+  },
+];
+
+const GitOpsDeploymentManager = (props) => {
+  const {
```

Comment:
moved to features/Gitops and converted to functional component
I have cut out a lot of code from this page and put it in context or in their own components
ie. SetupProvider and AppGitops